### PR TITLE
Add OpenStack and hive troubleshooting in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/24678

### CODE REVIEW

**Comment by oafischer** [comment]
@ldpliu I had to fill in some gaps because it wasn't always clear if a file or parameter was being referenced. What I have now may have errors so please check by comparing it to your original draft. Thanks.

File: troubleshooting/trouble_openstack_cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+name: txue-osspoke-openstack-certificatebundle
+----
++
+To create a secret with a matching file name, run the following command:
++
+```
+oc create secret generic txue-osspoke-openstack-certificatebundle --from-file=ca.crt=ca.crt.pem -n $CLUSTERNAME
+```
++
+. The size of the `ca.cert` file must be less than 63 thousand bytes.
```
Comment (dockerymick) [inline troubleshooting/trouble_openstack_cluster.adoc:54]: This looks good to me. We definitely need to revisit the architecture of our troubleshoot topics. I would remove _Managed cluster creation fails with unknown authority error_ from the subtitles. However, maybe keep it as is until we can receive feedback from Chris and Brandi

**Review by dockerymick — APPROVED** [review]
Left a comment

File: troubleshooting/trouble_openstack_cluster.adoc (hunk 1 above)
Comment (oafischer) [reply troubleshooting/trouble_openstack_cluster.adoc:54]: Thanks Mikela, sounds good. @swopebe and @cadawson, let us know what you think.

File: troubleshooting/trouble_openstack_cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Verify that the following files are configured correctly:
+
+. The `clouds.yaml` file must specify the path to the `ca.crt` file in the `cacert` parameter. The `cacert` parameter is passed to the OpenShift installer when generating the ignition shim. See the following example:
++
+[source,yaml]
+----
+clouds:
+openstack:
+cacert: "/etc/pki/ca-trust/source/anchors/ca.crt"
+----
```
Comment (ldpliu) [inline troubleshooting/trouble_openstack_cluster.adoc:30]: The code format should be updated follow: https://github.com/stolostron/backlog/issues/24678#issuecomment-1203433327

File: troubleshooting/trouble_openstack_cluster.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+baseDomain: dev09.red-chesterfield.com
+clusterName: txue-osspoke
+platform:
+openstack:
+cloud: openstack
+credentialsSecretRef:
+name: txue-osspoke-openstack-creds
+certificatesSecretRef:
+name: txue-osspoke-openstack-certificatebundle
+----
```
Comment (ldpliu) [inline troubleshooting/trouble_openstack_cluster.adoc:46]: The code format should be updated follow: https://github.com/stolostron/backlog/issues/24678#issuecomment-1203433327

**Review by ldpliu — APPROVED** [review]

**Review by ldpliu — APPROVED** [review]

File: troubleshooting/trouble_openstack_cluster.adoc [hunk 4]
```diff
@@ -0,0 +1,56 @@
+[#troubleshooting-cluster-creation-on-openstack]
+= Troubleshooting cluster creation on OpenStack
```
Comment (cadawson) [inline troubleshooting/trouble_openstack_cluster.adoc]: There is only one troubleshooting issue here, so I would remove this header and make "Managed cluster creation fails on Red Hat OpenStack Platform with unknown authority error" the main title. Then Symptom and Resolving the problem: become sub-levels. 
For example:
= Managed cluster creation fails on Red Hat OpenStack Platform with unknown authority error
== Symptom: Managed cluster creation fails on Red Hat OpenStack Platform with unknown authority error
== Resolving the problem: Managed cluster creation fails on Red Hat OpenStack Platform with unknown authority error

Let me know if that doesn't make sense.

File: troubleshooting/trouble_openstack_cluster.adoc [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+If you experience a problem when creating a {ocp} cluster on OpenStack, see the following troubleshooting information to see if one of them addresses your problem.
```
Comment (cadawson) [inline troubleshooting/trouble_openstack_cluster.adoc]: According to the product title guidance, we should always use "Red Hat OpenStack Platform".

**Review by cadawson — CHANGES_REQUESTED** [review]
Thanks @oafischer . Looks good. I added a couple of comments.

File: troubleshooting/trouble_openstack_cluster.adoc (hunk 4 above)
Comment (oafischer) [reply troubleshooting/trouble_openstack_cluster.adoc]: Thanks Chris! I went ahead and implemented your suggestions.

File: troubleshooting/trouble_openstack_cluster.adoc (hunk 1 above)
Comment (dockerymick) [reply troubleshooting/trouble_openstack_cluster.adoc:54]: Let's bring this up in the doc arch call after GA maybe? I would move forward with the merge

**Comment by oafischer** [comment]
Implemented suggested changes, moving forward with merge.

**Pull request merged**